src/services/fs.ts

```diff
@@ -184,12 +184,12 @@ export const getZipFileStream = async (
 };
 
 export async function isFolder(dirPath: string) {
-    return await fs
-        .stat(dirPath)
-        .then((stats) => {
-            return stats.isDirectory();
-        })
-        .catch(() => false);
+    try {
+        const stats = await fs.stat(dirPath);
+        return stats.isDirectory();
+    } catch (e) {
+        return false;
```

Comment:
what would is usually error? Can you also add a comment why we aren't logging exception as error or warn?

Reply:
So, this error's out when the path doesn't exist. (which can happen when a user has deleted the folder which he has added for watching)